src/java/org/apache/cassandra/cql3/statements/schema/AlterTableStatement.java

```diff
@@ -739,9 +739,9 @@ public KeyspaceMetadata apply(Epoch epoch, KeyspaceMetadata keyspace, TableMetad
     public static class AlterConstraints extends AlterTableStatement
```

Comment:
As with the grammar, although it isn't new to this patch (but it is obviously a whole new feature) I would suggest renaming this to `ConstrainColumn` to be more consistent with `MaskColumn/DropColumn/AddColumns/etc`

Reply:
Same as previous comment